src/emails/components/EmailDataPointCount.tsx

```diff
@@ -0,0 +1,126 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* c8 ignore start */
+
+import React from "react";
+import { ExtendedReactLocalization } from "../../app/functions/l10n";
+import { DashboardSummary } from "../../app/functions/server/dashboard";
+import { hasPremium } from "../../app/functions/universal/user";
+import { SubscriberRow } from "knex/types/tables";
+
+type Props = {
+  l10n: ExtendedReactLocalization;
+  utmCampaignId: string;
+  utmContentSuffix: string;
+  dataSummary: DashboardSummary;
+  subscriber: SubscriberRow;
+};
+
+export const DataPointCount = (props: Props) => {
+  const unresolvedDataBreaches = props.dataSummary.dataBreachUnresolvedNum;
+  const unresolvedDataBreachesAndBrokers =
+    props.dataSummary.dataBreachUnresolvedNum +
+    props.dataSummary.dataBrokerManuallyResolvedNum;
```

Comment:
Question: Is `unresolvedDataBreachesAndBrokers` supposed to include resolved manually resolved broker results?